README.md

```diff
@@ -51,43 +27,13 @@ const addOns: Array<blueprints.ClusterAddOn> = [
 const account = '<aws-account-id>';
 const region = '<aws-region>';
 const props = { env: { account, region } };
-new blueprints.EksBlueprint(app, { id: '<aws-eks-cluster-name>', addOns}, props);
+new blueprints.EksBlueprint(app, { id: '<aws-eks-cluster-name>', version: 'auto', addOns}, props);
 ```
 
-### Example Configuration (secrets in code):
-
-```typescript
-import 'source-map-support/register';
-import * as cdk from '@aws-cdk/core'
-import * as dt from '@dynatraceoss/dynatrace-eks-blueprints-addon'
-import * as blueprints from '@aws-quickstart/eks-blueprints'
-
-const app = new cdk.App();
-
-const Dynatrace = new dt.DynatraceAddOn({
-  apiUrl: "https://<your-dynatrace-tenant-url>/api",
-  apiToken: "<your-api-token",
-  paasToken: "<your-paas-token>"
-})
-
+### Create DynaKube
 
-const addOns: Array<blueprints.ClusterAddOn> = [
-    Dynatrace,
-];
-
-const account = '<aws-account-id>';
-const region = '<aws-region>';
-const props = { env: { account, region } };
-new blueprints.EksBlueprint(app, { id: '<aws-eks-cluster-name>', addOns}, props);
-```
-
-## Adding your EKS Cluster to Dynatrace
-After creating your EKS Cluster, connect to this cluster with kubectl and find out the UUID of the "kube-system" Namespace of this:
-```
-kubectl get ns kube-system -ojsonpath='{.metadata.uid}'
-```
+Create a DynaKube resource as described in the [README file of the Dynatrace Operator](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-k8s/reference/dynakube-parameters).
```

Comment:
I would expect an example of how to deploy the Custom Resource using the CDK in my project.